store/contentSchemas.json

```diff
@@ -0,0 +1,25 @@
+{
+  "definitions": {
+    "SwitcherListItem": {
+      "title": "admin/editor.locale-switcher.title",
+      "properties": {
+        "display": {
+          "title": "admin/editor.locale-switcher.display.title",
+          "description": "admin/editor.locale-switcher.display.description",
+          "type": "string",
+          "enum": ["default", "none"],
```

Comment:
I don't know why, but this is not changing, the workspace has a console.log to show the result that gets to the component